packages/rollup-contracts/package.json

```diff
@@ -13,7 +13,8 @@
     "fix": "prettier --config ../../prettier-config.json --write \"index.ts\" \"{deploy,test}/**/*.ts\"",
     "build": "waffle waffle-config.json && tsc -p .",
     "clean": "rimraf build/",
-    "deploy:rollup-chain": "yarn build && node ./build/deploy/rollup-chain.js"
+    "deploy:rollup-chain": "yarn build && node ./build/deploy/rollup-chain.js",
+    "womp": "mocha --require ts-node/register 'test/merklization/EthMerkleTrie.spec.ts' --timeout 20000"
```

Comment:
Lol is `womp` a debugging thing that you forgot to remove?

Reply:
Hahaha yes, whoops

Reply:
Fixed